### ISSUE TITLE
Before sending iterations >=2, try to fetch the tags

### ISSUE THREAD

**Issue by dscho** [gitgitgadget/gitgitgadget#34]
To generate the range-diff, the tag of the previous iteration needs to be present. So we need to fetch it.

See https://github.com/gitgitgadget/git/pull/33#issuecomment-418529313 for an example where we failed to have all necessary objects in the local mirror.

**Comment by dscho** [comment]
This is curious: it should have been addressed by 15183de9c332c4c1529169c9e2e013f3e8cec3c6.

The problem occurred mainly because the tag was created outside GitGitGadget (I manually created the metadata so that I could submit a v2 of a patch series that started outside GitGitGadget).

Let's see if that happens, still, with https://github.com/gitgitgadget/git/pull/101. If it does, at least now we will have a log (since we switched to an Azure Function/Pipelines combo).
